fvm/env.go

```diff
@@ -495,6 +537,14 @@ func (e *transactionEnv) GetSigningAccounts() []runtime.Address {
 	return e.authorizers
 }
 
+func (e *transactionEnv) TxIndex() uint32 {
+	return e.txIndex
+}
+
+func (e *transactionEnv) TxID() flow.Identifier {
+	return e.txID
+}
```

Comment:
Are these required by Cadence now?

Reply:
not needed by Cadence but useful for tests